src/lib/reasoners/adt_rel.ml

```diff
@@ -36,48 +36,68 @@ module LR = Uf.LX
 module Th = Shostak.Adt
 module SLR = Set.Make(LR)
 
-module TSet = Set.Make
-    (struct
-      type t = Uid.t
-
-      (* We use a dedicated total order on the constructors to ensure
-         the termination of the model generation. *)
-      let compare = Nest.compare
-    end)
+module DE = Dolmen.Std.Expr
+module DT = Dolmen.Std.Expr.Ty
+module B = Dolmen.Std.Builtin
+module TSet = Set.Make (Int)
```

Comment:
This is no longer a set of types — `Hset`, or even `SI`, would be better.

Nit: there is `Util.SI` :upside_down_face:

Reply:
Sure